test/unit/test-kernel.cpp

```diff
@@ -679,6 +679,47 @@ TEST(Kernel, FissionFusion_Conditional)
   delete[] x;
 }
 
+
```

Comment:
We should also have Count tests for RangeStrideSegment, ListSegment, and IndexSet with multiple segments and segment types. When JIRA comes back on line, I will add an issue for our test reorg...

Reply:
The cuda backend will need tests for each valid cuda policy combination.